src/test/java/core/basesyntax/strategy/handler/impl/ReturnHandlerTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.strategy.handler.impl;
+
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.handler.OperationHandler;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReturnHandlerTest {
+
+    private static final String APPLE = "apple";
+    private static final int VALID_FRUIT_QUANTITY = 5;
+    private static final String BANANA = "banana";
+    private static final int NEGATIVE_FRUIT_QUANTITY = -7;
+    private static final int ZERO_FRUIT_QUANTITY = 0;
+    private static final int ADD_FRUIT_QUANTITY = 7;
+    private static OperationHandler handler;
+
+    @BeforeAll
+    public static void beforeAll() {
+        handler = new ReturnHandler();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        Storage.FRUITS.put(BANANA, VALID_FRUIT_QUANTITY);
+    }
+
+    @AfterEach
+    public void afterEach() {
+        Storage.FRUITS.clear();
+    }
+
+    @Test
+    public void test_Add_To_Empty_FruitStorage_ok() {
+        Storage.FRUITS.clear();
```

Comment:
It's redundand code, you can just delete it, because you clean storage after every test